src/commands/watch.rs

```diff
@@ -119,7 +111,10 @@ impl WatchCommand {
                     return Ok::<(), Infallible>(());
                 }
 
-                // TODO: Check if spin.toml changed and reconfigure
+                if event.paths().any(|(p, _)| p.ends_with("spin.toml")) {
```

Comment:
You should know the path of the manifest; use that instead of the literal text "spin.toml"